doc/manual.md

```diff
@@ -358,21 +358,40 @@ local x1 : integer = ns[1]
 local x2 : integer = ns[4] -- run-time error
 ```
 
+## Exporting top-level components
+
+The `export` keyword can be used to export top-level components such as variables
+and functions from a Pallene module. The keyword `export` can be used in the
+same places where the `local` keyword can be used. It should be noted that, every
+top-level component declaration should have either `local` or `export` modifier.
```

Comment:
To help keep the git diffs easy to follow, we try to put each sentence on a separate line. It is OK if the line is longer than 80 characters. Our editor config should have disabled to max_line_length for markdown files, let me know if that isn't working for you.